### ISSUE TITLE
fix how to guides that jump from h1 to h3 headings

### ISSUE THREAD

**Issue by anita-trimbur** [pulumi/registry#1940]
how-to-guide sub-pages skip heading levels from h1 to h3 which poses an accessibility issue for folks using screen readers. tags that are h3 now should be shifted up to a h2 and so forth

<img width="726" alt="Screen Shot 2023-01-23 at 2 27 34 PM" src="https://user-images.githubusercontent.com/97129308/214131539-9422cffb-84c0-47dc-ab7d-9fc798dc0cb6.png">
